x-pack/test/cloud_security_posture_functional/helpers.ts

```diff
@@ -4,24 +4,38 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
 
-const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_NAME = 'cloud_security_posture';
+const CLOUD_SECURITY_PACKAGE_VERSION = '1.0.8';
 
 /**
- * flags to load kibana with fleet pre-configured to have 'cloud_security_posture' integration installed
+ * Package version is fixed (not latest) so FTR won't suddenly break when package is changed.
+ *
+ * test a new package:
+ * 1. merge a new package to EPR
+ * 2. locally checkout the kibana version that matches the new package
+ * 3. update CLOUD_SECURITY_PACKAGE_VERSION with the new package version
+ * 4. run FTR test locally
+ * 5. when test pass, merge new kibana changes with the new package version.
+ *
  */
-export const getPreConfiguredFleetPackages = () => [
-  `--xpack.fleet.packages.0.name=${CLOUD_SECURITY_POSTURE_PACKAGE_NAME}`,
-  `--xpack.fleet.packages.0.version=latest`,
+const getCloudSecurityPackageConfig = () => [
+  `--xpack.fleet.packages.0.name=${CLOUD_SECURITY_PACKAGE_NAME}`,
+  `--xpack.fleet.packages.0.version=${CLOUD_SECURITY_PACKAGE_VERSION}`,
```

Comment:
this is the main change in this PR. it locks the version we use in CI for our FTR tests. rest of changes are just renames/moving stuff around.